include/grpc/grpc.h

```diff
@@ -423,6 +423,12 @@ grpc_call *grpc_channel_create_registered_call(
 grpc_call_error grpc_call_start_batch(grpc_call *call, const grpc_op *ops,
                                       size_t nops, void *tag);
 
+/* Returns a newly allocated string representing the endpoint to which this
+   call is communicating with. The string is in the uri format accepted by
+   grpc_channel_create.
+   The returned string should be disposed of with gpr_free(). */
```

Comment:
Can you please add that this function should not be used for any security related decision process as the peer information returned in this function is not authenticated. 
The auth_context should be used instead for the security/authorization decisions.